.github/workflows/ci.yml

```diff
@@ -57,3 +59,27 @@ jobs:
         env:
           CI_NAME: github-actions
           COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+      - name: Pack
+        run: dotnet pack src -c Release --no-build --output nuget
+      - name: 'Upload Artifact'
+        uses: actions/upload-artifact@v2
+        with:
+          name: nuget
+          path: nuget/
+
+  publish:
+    if: (github.event_name == 'workflow_dispatch') && (github.ref == 'refs/heads/master') && (endsWith(github.actor, '-stripe')
```

Comment:
Do we want to set up a CODEOWNERS file in .github as well to limit modification to publishing?